events/sick_man_events.txt

```diff
@@ -0,0 +1,664 @@
+			is_decaying = yes
+		}
+		add_modifier = {
+			name = sick_man_liberal_western_education
+		}
+		change_relations = {
+			country = c:FRA
+			value = 30
+		}
+		change_relations = {
+			country = c:GBR
+			value = 30
+		}
+		change_relations = {
+			country = c:PRU
+			value = 30
+		}
+		change_relations = {
+			country = c:USA
+			value = 30
+		}
+	}
+	option = {
+		name = sick_man.2.b
+		default_option = yes
+		add_loyalists = {
+			value = 0.05
+			country = c:TUR
+		}
+		add_modifier = {
+			name = sick_man_liberal_constitution_new
+			months = 180
+		}
+		add_modifier = {
+			name = sick_man_liberal_constitution_sultan
+		}
+		add_primary_culture = cu:misri
+	}
+	option = {
+		name = sick_man.2.c
+		default_option = yes
+		# Change to Religion-based when Paradox Fixes
+		add_loyalists = {
+			value = 0.15
+			culture = cu:albanian
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:bosniak
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:greek
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:bosniak
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:bulgarian
+		}
+		add_loyalists = {
+			value = 0.10
+			culture = cu:bedouin
+		}
+		add_loyalists = {
+			value = 0.10
+			culture = cu:kurdish
+		}
+		add_loyalists = {
+			value = 0.10
+			culture = cu:misri
+		}
+		add_modifier = {
+			name = sick_man_liberal_crossroads_boost
+			months = 120
+		}
+		add_modifier = {
+			name = sick_man_liberal_crossroads_migration
+		}
+		add_primary_culture = cu:greek
+	}
+	option = {
+		name = sick_man.2.d
+		default_option = yes
+		# Change to Religion-based when Paradox Fixes
+		add_loyalists = {
+			value = 0.15
+			culture = cu:turkish
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:kurdish
+		}
+		add_loyalists = {
+			value = 0.15
+			culture = cu:azerbaijani
+		}
+		add_modifier = {
+			name = sick_man_liberal_turanian_heritage
+			months = 180
+			is_decaying = yes
+		}
+		add_modifier = {
+			name = sick_man_liberal_turanian_movement
+		}
+		add_primary_culture = cu:turkmen
+	}
+
+}
+
+# Fail event
+sick_man.3 = {
+	type = country_event
+	placement = root
+	title = sick_man.3.t
+	desc = sick_man.3.d
+	flavor = sick_man.3.f
+	icon = "gfx/interface/icons/event_icons/event_skull.dds"
+	duration = 3
+	event_image = {
+		video = "unspecific_factory_closed"
+	}
+	on_created_soundeffect = "event:/SFX/UI/Alerts/event_appear"
+	trigger = {
+	}
+	immediate = {
+		set_variable = sick_man_complete
+		ruler = {
+			save_scope_as = ottoman_sultan
+		}
+	}
+	option = {
+		name = sick_man.3.a
+		if = {
+			limit = {
+				is_subject = no
+				any_subject_or_below = {
+					NOT = {
+						any_overlord_or_above = {
+							NOT = {
+								ROOT = this
+							}
+						}
+					}
+					NOT = {
+						is_country_type = unrecognized
+					}
+				}
+			}
+			every_subject_or_below = {
+				limit = {
+					NOT = {
+						any_overlord_or_above = {
+							NOT = {
+								ROOT = this
+							}
+						}
+					}
```

Comment:
Very complicated way to write that it needs to be a direct subject lol.

Reply:
This is also a paradox moment. Let me know if you have a better way to clean it up

Reply:
`is_direct_subject_of = c:CHI` would work.